webcompat/static/js/lib/diagnose.js

```diff
@@ -5,7 +5,7 @@
 var diagnose = diagnose || {}; // eslint-disable-line no-use-before-define
 var issues = issues || {}; // eslint-disable-line no-use-before-define
 
-diagnose.NewCollection = Backbone.Collection.extend({
+diagnose.NeedsTriageCollection = Backbone.Collection.extend({
   model: issues.Issue,
   url: '/api/issues/category/needstriage'
 });
```

Comment:
Looks good! Just a few more places to change:

https://github.com/haseebgit/webcompat.com/blob/6e5a553850968d89eb429d6c454e22a49c6d4bf4/webcompat/static/js/lib/diagnose.js#L13

https://github.com/haseebgit/webcompat.com/blob/6e5a553850968d89eb429d6c454e22a49c6d4bf4/webcompat/static/js/lib/diagnose.js#L18

https://github.com/haseebgit/webcompat.com/blob/6e5a553850968d89eb429d6c454e22a49c6d4bf4/webcompat/static/js/lib/diagnose.js#L34

https://github.com/haseebgit/webcompat.com/blob/6e5a553850968d89eb429d6c454e22a49c6d4bf4/webcompat/static/js/lib/diagnose.js#L23

https://github.com/haseebgit/webcompat.com/blob/6e5a553850968d89eb429d6c454e22a49c6d4bf4/webcompat/templates/home-page/browse-issues.html#L5